lib/options-manager.js

```diff
@@ -322,6 +383,27 @@ const mergeWithPrettierConfig = (options, prettierOptions) => {
 	);
 };
 
+const buildTSConfig = options => config => {
+	if (options.ts) {
+		config.baseConfig.extends = config.baseConfig.extends.concat('xo-typescript');
+		config.baseConfig.parser = '@typescript-eslint/parser';
```

Comment:
This is already defined by `eslint-config-xo-typescript`.

And there we resolve it to its absolute path, not sure if that makes a difference though: https://github.com/xojs/eslint-config-xo-typescript/blob/99c4590d599933f754c498c02cf95cec3c6d04a1/index.js#L3

Reply:
Done